lib/sidekiq/cron/poller.rb

```diff
@@ -7,16 +7,6 @@ module Sidekiq
   module Cron
     # The Poller checks Redis every N seconds for sheduled cron jobs.
     class Poller < Sidekiq::Scheduled::Poller
-      def initialize(options = {})
```

Comment:
the parameter is a `Sidekiq::Config` in 7. The inherited method's parameter is non-optional, so should probably bring this back, also to keep compatibility with before-6.5